src/cart/crt_iv.c

```diff
@@ -1749,6 +1884,14 @@ crt_hdlr_iv_sync_aux(void *arg)
 			D_GOTO(exit, rc = -DER_NONEXIST);
 	}
 
+	/* Check group version match */
+	grp_ver = ivns_internal->cii_grp_priv->gp_membs_ver;
+	if (grp_ver != input->ivs_grp_ver) {
+		D_ERROR("Group (%s) version mismatch. Local: %d Remote :%d\n",
+			ivns_id.ii_group_name, grp_ver,
+			input->ivs_grp_ver);
+		D_GOTO(exit, rc = -DER_GRPVER);
+	}
```

Comment:
i dont think this check is needed as we dont compute parent and/or children in this function

Reply:
This is a comparison between an incoming rpc version and the node's group version.  I don't know if a check at this point is valid, but don't want to rule it out simply because we are not evaluating a parent/child.

I'll leave that decision up to you.